pyodide-build/pyodide_build/pywasmcross.py

```diff
@@ -390,11 +389,8 @@ def _calculate_object_exports_readobj_parse(output: str) -> list[str]:
 
 
 def calculate_object_exports_readobj(objects: list[str]) -> list[str] | None:
-    which_emcc = shutil.which("emcc")
-    assert which_emcc
-    emcc = Path(which_emcc)
     args = [
-        str((emcc / "../../bin/llvm-readobj").resolve()),
+        "llvm-readobj",
```

Comment:
Will this work for out-of-tree build where users install emscripten by themselves?

Reply:
Ah! Good catch. We can put this back in as a fallback if it's not on the path.